test/browser.js

```diff
@@ -176,3 +178,56 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 
 	await server.close();
 });
+
+test('FormData with searchParams', withPage, async (t, page) => {
+	const server = await createTestServer();
+	server.get('/', (request, response) => {
+		response.end('nothing');
+	});
+	server.post('/', async (request, response) => {
+		const pBody = util.promisify(body);
+		response.end(await pBody(request));
+	});
+
+	await page.goto(server.url);
+	await page.addScriptTag({path: './umd.js'});
+
+	const requestBody = await page.evaluate(url => {
+		window.ky = window.ky.default;
+		const formData = new window.FormData();
+		formData.append('file', new window.File(['bubblegum pie'], 'my-file'));
+		return window.ky(url, {
+			method: 'post',
+			searchParams: 'foo=1',
```

Comment:
The test should assert that the search params are respected

Reply:
I added an assertion for the search params and moved the assertions into the server callback.